### PR TITLE
updated README to match wca-live

### PR DESCRIPTION
**Pull request by dunkOnIT**
[WCA Live](https://github.com/thewca/wca-live/)'s README looks great, updated ours to match the styling in the heading section

### CODE REVIEW

**Comment by dunkOnIT** [comment]
Failure look unrelated - re-running tests to confirm they aren't intermittent.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
-# worldcubeassociation.org [![Build Status](https://github.com/thewca/worldcubeassociation.org/actions/workflows/ruby.yml/badge.svg?event=push)](https://github.com/thewca/worldcubeassociation.org/actions/workflows/ruby.yml) [![Coverage Status](https://coveralls.io/repos/github/thewca/worldcubeassociation.org/badge.svg?branch=master)](https://coveralls.io/github/thewca/worldcubeassociation.org?branch=master)
+<h1 align="center">WorldCubeAssociation Website</h1>
+<div align="center">
+  <img height="120" src="client/public/favicon.png" />
```
Comment (gregorbg) [inline README.md:3]: This file doesn't exist in our repo

File: README.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+</div>
+<br />
+<div align="center">
+  <strong>
+    This repository contains all of the code that runs on [worldcubeassociation.org](https://www.worldcubeassociation.org/).
+  </strong>
+</div>
+<br />
+<div align="center">
```
Comment (gregorbg) [inline README.md:12]: You're opening a `div` here but you never close it

**Review by gregorbg — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dunkOnIT) [reply README.md:12]: Div now closes

**Comment by dunkOnIT** [comment]
Sorry about the silly mistakes - got a .md renderer going now.

File: README.md (hunk 1 above)
Comment (dunkOnIT) [reply README.md]: Downloaded the png from WCA Live and added it to our repo - looked at the existing files we have, but all the logos have "World Cube Association" at the bottom, and I don't believe .md files can render .ico, which is why I didn't use our favicon.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,18 @@
-# worldcubeassociation.org [![Build Status](https://github.com/thewca/worldcubeassociation.org/actions/workflows/ruby.yml/badge.svg?event=push)](https://github.com/thewca/worldcubeassociation.org/actions/workflows/ruby.yml) [![Coverage Status](https://coveralls.io/repos/github/thewca/worldcubeassociation.org/badge.svg?branch=master)](https://coveralls.io/github/thewca/worldcubeassociation.org?branch=master)
-
-This repository contains all of the code that runs on [worldcubeassociation.org](https://www.worldcubeassociation.org/).
+<h1 align="center">WorldCubeAssociation Website</h1>
+<div align="center">
+  <img height="120" src="WcaOnRails/public/" />
```
Comment (gregorbg) [inline README.md:3]: image source path is still wrong

**Review by gregorbg — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (dunkOnIT) [reply README.md]: Corrected

**Review by gregorbg — APPROVED** [review]

**Pull request merged**